proto/ibc/core/client/v1/client.proto

```diff
@@ -57,9 +57,9 @@ message ClientUpdateProposal {
 // UpgradeProposal is a gov Content type for initiating an IBC breaking
 // upgrade.
 message UpgradeProposal {
-  option (gogoproto.goproto_getters)         = false;
-  option (gogoproto.goproto_stringer)        = false;
-  option (gogoproto.equal)                   = true;
```

Comment:
this one is true, is it required or fine to remove?

Reply:
agh. i didnt intend to remove this one actually. i'll add it back

Reply:
I don't see any usages of its equal function, but also curious!

Reply:
I don't see why we would need to check for equality on upgrade proposals, it makes sense for `Coin` tho